Firestore/Example/Firestore.xcodeproj/project.pbxproj

```diff
@@ -2411,7 +2411,7 @@
 				"${PODS_ROOT}/Target Support Files/Pods-Firestore_Example_tvOS/Pods-Firestore_Example_tvOS-frameworks.sh",
 				"${BUILT_PRODUCTS_DIR}/BoringSSL-GRPC-tvOS/openssl_grpc.framework",
 				"${BUILT_PRODUCTS_DIR}/GTMSessionFetcher-tvOS/GTMSessionFetcher.framework",
-				"${BUILT_PRODUCTS_DIR}/GoogleUtilities-Environment-Logger-tvOS/GoogleUtilities.framework",
+				"${BUILT_PRODUCTS_DIR}/GoogleUtilities-85d94889/GoogleUtilities.framework",
```

Comment:
I don't know what caused this diff, but presumably it's intentional / correct...?

Reply:
The latest changes to Firebase 5.20/5.20.1 cause this when you `pod install`. I'm just committing the results of that.